src/pages/api/sysinfo.ts

```diff
@@ -10,5 +10,7 @@ export default function handler(req: NextApiRequest, res: NextApiResponse) {
     version: __VERSION__,
     buildId: __BUILD_ID__,
     nodeEnv: __NODE_ENV__,
+    analyticsUrl: process.env.MATOMO_URL,
```

Comment:
@abbyoung this info seemed appropriate to add to this existing endpoint, but let me know if you feel otherwise!!